Code/Tools/ProjectManager/Platform/Linux/ProjectUtils_linux.cpp

```diff
@@ -26,32 +26,76 @@ namespace O3DE::ProjectManager
             return AZ::Success();
         }
 
-        AZ::Outcome<QString, QString> FindSupportedCompilerForPlatform()
+        AZ::Outcome<QString, QString> FindSupportedCMake()
         {
             // Validate that cmake is installed and is in the command line
-            auto whichCMakeResult = ProjectUtils::ExecuteCommandResult("which", QStringList{ProjectCMakeCommand});
+            auto whichCMakeResult = ProjectUtils::ExecuteCommandResult("which", QStringList{ ProjectCMakeCommand });
             if (!whichCMakeResult.IsSuccess())
             {
-                return AZ::Failure(QObject::tr("CMake not found. <br><br>"
-                    "Make sure that the minimum version of CMake is installed and available from the command prompt. "
-                    "Refer to the <a href='https://o3de.org/docs/welcome-guide/setup/requirements/#cmake'>O3DE requirements</a> page for more information."));
+                return AZ::Failure(
+                    QObject::tr("CMake not found. <br><br>"
+                                "Make sure that the minimum version of CMake is installed and available from the command prompt. "
+                                "Refer to the <a href='https://o3de.org/docs/welcome-guide/setup/requirements/#cmake'>O3DE "
+                                "requirements</a> page for more information."));
+            }
+
+            QString cmakeInstalledPath = whichCMakeResult.GetValue().split("\n")[0];
+
+            // Query the version of the installed cmake
+            auto queryCmakeVersionQuery = ExecuteCommandResult(cmakeInstalledPath, QStringList{ "--version" });
+            if (queryCmakeVersionQuery.IsSuccess())
+            {
+                AZ_TracePrintf(
+                    "Project Manager", "\"%s\" detected.", queryCmakeVersionQuery.GetValue().split("\n")[0].toUtf8().constData());
+            }
+
+            return AZ::Success(QString{ cmakeInstalledPath });
```

Comment:
The header states: `//! Detect if cmake is installed and is the minimum version for O3DE` but it doesn't seem to be checking the returned version string for a minimum version.

Should a version check be added here?  Ideally we keep the real minimum version check in one place, the one in root `CMakeFiles.txt`.  Having multiple locations where the minimum version is defined (cmake and cpp) could become out of sync.

If we did check versions here, could change this to `which -a cmake` and loop over the results, just in case someone has multiple on their system.

Reply:
Well it wasn't detecting the version before, but I added the comment to reserve the right for ProjectManager to perform the cmake version detection in the future.

That being said I believe that cmake is more than capable of detecting it's version and instead ProjectManager should focus on managing O3DE Projects, not cmake nor compilers